modules/db/src/main/java/org/mule/module/db/internal/domain/connection/OracleDbConnection.java

```diff
@@ -122,9 +157,9 @@ protected void resolveLobs(String typeName, Object[] attributes) throws SQLExcep
         }
     }
 
-    private Map<Integer, String> getDataTypes(String typeName) throws SQLException
+    private Map<Integer, String> getLobFieldsDataTypeInfo(String typeName) throws SQLException
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Rename this method; there is a "private" method in the parent class with the same name. ([squid:S2177](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS2177))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7681&issues=AWjyIkAFotIXnmsUSQ_c&open=AWjyIkAFotIXnmsUSQ_c)

[ISSUE_KEY]: AWjyIkAFotIXnmsUSQ_c